docs/atlascli/command/atlas-backups-exports-buckets-create.txt

```diff
@@ -0,0 +1,95 @@
+.. _atlas-backups-exports-buckets-create:
+
+====================================
+atlas backups exports buckets create
+====================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Create new export bucket.
```

Comment:
```suggestion
Create an export destination for Atlas backups using an existing AWS S3 bucket.
```
Maybe not the most elegant solution, but I think it will be less confusing for customers and still uses "create" vs "configure" or "link"